scripts/integration-test.sh

```diff
@@ -1,14 +1,15 @@
 #!/bin/bash
 SRC_PATH=$1
 CUSTOM_COMMAND="${2:-test}"
-GATSBY_PATH="/home/circleci/project"
+GATSBY_PATH="${CIRCLE_WORKING_DIRECTORY:-../../}"
 
 sudo npm install -g gatsby-dev-cli &&
 
 # setting up child integration test link to gatsby packages
 cd $SRC_PATH &&
 yarn &&
 gatsby-dev --set-path-to-repo $GATSBY_PATH &&
-gatsby-dev --scan-once --copy-all && # copies _all_ files in gatsby/packages
+gatsby-dev --scan-once --copy-all --quiet && # copies _all_ files in gatsby/packages
+sudo chmod +x ./node_modules/.bin/gatsby && # this is sometimes necessary to ensure executable
```

Comment:
Note: I should maybe make this change at the gatsby-dev-cli level, instead, but this works for now.

Does anyone else run into this? I can replicate it pretty reliably on my local machine 🙃

Reply:
huh really? I've never hit that problem — I thought NPM took care of that

Reply:
Yeah, it's replicated in that test I linked to too to the best of my knowledge. Weird stuff 😱